### PR TITLE
ENH: add lower triangular scalars from DTI (could be useful for ML)?

### PR DESCRIPTION
**Pull request by 36000**
These can already be calculated from our derivatives, but now they are formally added to the task system. 
Also adds a change for tract profile calculation that should significantly speed it up.

### CODE REVIEW

File: AFQ/tasks/segmentation.py [hunk 1]
```diff
@@ -304,11 +304,16 @@ def tract_profiles(clean_bundles, data_imap,
         this_sl = seg_sft.get_bundle(bundle_name).streamlines
         if len(this_sl) == 0:
             continue
+        if profile_weights == "gauss":
+            # calculate only once per bundle
+            bundle_profile_weights = gaussian_weights(this_sl)
```
Comment (arokem) [inline AFQ/tasks/segmentation.py:309]: Good call!

File: AFQ/tasks/data.py [hunk 2]
```diff
@@ -280,6 +280,24 @@ def dti_fa(dti_tf):
     return dti_tf.fa
 
 
+@pimms.calc("dti_lt0", "dti_lt1", "dti_lt2", "dti_lt3", "dti_lt4", "dti_lt5")
```
Comment (arokem) [inline AFQ/tasks/data.py:283]: Do we want `dki_lt` elements as well? I believe there are 15 elements in DKI. 

And relatedly, we often calculate FA/MD from DKI, when that's doable (i.e., when there are multiple non-zero b-values). Do we want to be able to extract the six DTI elements from DKI?

**Comment by 36000** [comment]
@arokem this might be useful for the final NKI run.

**Review by arokem — COMMENTED** [review]
Definitely an improvement! I have some questions about how we might incorporate DKI in here as well.

**Comment by arokem** [comment]
> @arokem this might be useful for the final NKI run.

Agreed! And the DKI questions do not pertain to that, so we could merge this as-is for that, and then come back to deal with those questions later on.

**Comment by 36000** [comment]
@arokem  can we merge this and do DKI in a separate PR?

**Pull request merged**